plugins/woocommerce/includes/wc-page-functions.php

```diff
@@ -133,8 +133,8 @@ function wc_nav_menu_items( $items ) {
 				if ( empty( $item->url ) ) {
 					continue;
 				}
-				$path  = wp_parse_url( $item->url, PHP_URL_PATH );
-				$query = wp_parse_url( $item->url, PHP_URL_QUERY );
+				$path  = (string)wp_parse_url( $item->url, PHP_URL_PATH );
```

Comment:
Using a null coelesce will work here

```suggestion
				$path  = wp_parse_url( $item->url, PHP_URL_PATH ) ?? '';
```